Research/ResearchUI/iOS/Step View Controllers/RSDCountdownStepViewController.swift

```diff
@@ -101,6 +101,7 @@ open class RSDCountdownStepViewController: RSDFullscreenImageStepViewController
         super.setColorStyle(for: placement, background: background)
         if placement == .body {
             countdownLabel?.textColor = countdownLabelColor(on: background)
+            self.countdownLabel?.font = self.designSystem.fontRules.font(for: .counter)
```

Comment:
These are currently statically defined b/c as graphical elements, they need to fit the view. To allow for customization of the font family, instead they should use the `font(forSize:weight:)` method described above.